src/calculateRentalCost.js

```diff
@@ -5,6 +5,20 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  let price = days * PRICE_PER_DAY;
```

Comment:
Consider using `const` instead of `let` for the `price` variable since it is not being reassigned, which adheres to the checklist item 3 that suggests preferring `const` over `let` where possible to avoid unintentional changes.